packages/components/utils/src/Portal/Portal.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Portal'
+type: 'component'
+slug: /utils/portal/
+typescript: ./Portal.tsx
+v4Doc: true
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+`Portal` component can be used to render children in a DOM node outside of its parent component. It can be useful for rendering components that you don’t want to be clipped by their parent’s component. For example: modals, popovers, dropdown, etc.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+- [Props (API reference)](#props-api-reference)
+
+## Import
+
+```jsx static=true
+import { Portal } from '@contentful/f36-utils';
+```
+
+## Example
+
+### Basic usage
+
+```jsx
+<Flex>
+  <Portal>
+    <Flex
+      style={{
+        backgroundColor: 'gray',
+      }}
+    >
+      Hi! I'm first in the document flow, but will be rendered at the end of the
+      document!
+    </Flex>
+  </Portal>
+  Parent component content
+</Flex>
+```
+
+## Props (API reference)
```

Comment:
I think Portal has a story in storybook, no?

Isn't this the same Portal that this doc is about?
https://v4-f36-storybook.netlify.app/?path=/story/utilities-portal

Reply:
yes, you are right!